paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -688,33 +679,28 @@ def autoscale_marathon_instance(
                 safe_downscaling_threshold=safe_downscaling_threshold,
                 task_data_insufficient=task_data_insufficient,
             )
-            if new_instance_count != current_instances:
-                if new_instance_count < current_instances and task_data_insufficient:
-                    write_to_log(
-                        config=marathon_service_config,
-                        line="Delaying scaling *down* as we found too few healthy tasks running in marathon. "
-                        "This can happen because tasks are delayed/waiting/unhealthy or because we are "
-                        "waiting for tasks to be killed. Will wait for sufficient healthy tasks before "
-                        "we make a decision to scale down.",
-                        level="debug",
-                    )
-                    return
-                else:
-                    set_instances_for_marathon_service(
-                        service=marathon_service_config.service,
-                        instance=marathon_service_config.instance,
-                        instance_count=new_instance_count,
-                    )
-                    write_to_log(
-                        config=marathon_service_config,
-                        line="Scaling from %d to %d instances (%s)"
-                        % (
-                            current_instances,
-                            new_instance_count,
-                            humanize_error(error),
-                        ),
-                        level="event",
-                    )
+            if new_instance_count < current_instances and task_data_insufficient:
+                write_to_log(
+                    config=marathon_service_config,
+                    line="Delaying scaling *down* as we found too few healthy tasks running in marathon. "
+                    "This can happen because tasks are delayed/waiting/unhealthy or because we are "
+                    "waiting for tasks to be killed. Will wait for sufficient healthy tasks before "
+                    "we make a decision to scale down.",
+                    level="debug",
+                )
+                return
+            else:
```

Comment:
Won't this create a lot of excess noise when the old/new are the same?

Reply:
Fixed.